src/course-outline/CourseOutline.jsx

```diff
@@ -40,6 +52,21 @@ const CourseOutline = ({ courseId }) => {
     <>
       <Container size="xl" className="m-4">
         <section className="course-outline-container mb-4 mt-5">
+          <TransitionReplace>
+            {showSuccessAlert ? (
+              <AlertMessage
+                key={intl.formatMessage(messages.alertSuccessAriaLabelledby)}
+                show={showSuccessAlert}
+                variant="success"
+                icon={CheckCircleIcon}
+                title={intl.formatMessage(messages.alertSuccessTitle)}
+                description={intl.formatMessage(messages.alertSuccessDescription)}
+                aria-hidden="true"
+                aria-labelledby={intl.formatMessage(messages.alertSuccessAriaLabelledby)}
+                aria-describedby={intl.formatMessage(messages.alertSuccessAriaDescribedby)}
+              />
+            ) : null}
```

Comment:
Just wondering: why not `bool && <component>` instead of `bool ? <component> : null`?
Is there a chance `showSuccessAlert` may end up into something renderable?

Reply:
Done